lib/arc/storage/local.ex

```diff
@@ -16,10 +16,10 @@ defmodule Arc.Storage.Local do
   def url(definition, version, file_and_scope, _options \\ []) do
     local_path = build_local_path(definition, version, file_and_scope)
 
-    if String.starts_with?(local_path, "/") do
-      local_path
+    if host() == "" do
+      Path.join "/", local_path
     else
-      "/" <> local_path
+      Path.join [host(), "/", local_path]
```

Comment:
This line breaks my tests as it creates URLs like `"http://static.example.com/./arctest/uploads/original-binary%20file.png"`

Should be `Path.join [host, local_path]`